### DISCUSSION TITLE
Regex for sbt manager is not considering packages under project/Dependencies.scala

### DISCUSSION THREAD

**Discussion by praveenreddy3621** [renovatebot/renovate#9343] [Request Help]
Hi,

The default file matching for sbt: https://docs.renovatebot.com/modules/manager/sbt/
with regex `project/[^/]*.scala$` is not considering my `Dependencies.scala `file under project folder

While I am running locally it considering `plugins.sbt` file under projects but could not consider the file under `project/Dependencies.scala` matching the regex.

Logs shows that it is listing all the files matching that regex which includes `project/Dependencies.scala` but while patching it is only considering the plugins.sbt under sbt manger for patching.

How can we fix this?

**Comment by rarkins** [comment]
Please create a [minimum reproduction](https://github.com/renovatebot/renovate/blob/master/docs/development/minimal-reproductions.md) to demonstrate the problem
